corehq/apps/reports/standard/project_health.py

```diff
@@ -184,12 +241,28 @@ def show_in_navigation(cls, domain=None, project=None, user=None):
     def bootstrap3_dispatcher(self, request, *args, **kwargs):
         super(ProjectHealthDashboard, self).bootstrap3_dispatcher(request, *args, **kwargs)
 
+    def get_filtered_group_ids(self):
+        return self.request.GET.getlist('group')
+
+    def get_users_by_filtered_groupids(self):
+        groupids_param = self.get_filtered_group_ids()
+        users_list = GroupES().domain('ccdt').group_ids(groupids_param).fields(["users"]).values()
+        user_id_list = []
+        for user in users_list:
+            usersid = user.values()[0]
+            if type(usersid) is list:
```

Comment:
If the group has more than one user, then I would expect usersid to be a list, and a string if the group only has one user. I definitely should look out for inheritance, so thank you for pointing out `isinstance` function!